xclim/indices/_agro.py

```diff
@@ -884,6 +885,159 @@ def water_budget(
     return out
 
 
+@declare_units(
+    pr="[precipitation]",
+    s_thresh_wet="[length]",
+    s_thresh_dry="[length]",
+    e_thresh_dry="[length]",
+)
+def rain_season(
```

Comment:
This might be a big ask, but it would be nice to have a `note` with the breakdown of the formulae involved here.

Would `thresh_wet_start` be better than `s_thresh_wet`?